src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -1003,6 +1003,40 @@ private void handleRawBatchPut(RawBatchPutResponse resp) {
     return resp.getKvsList();
   }
 
+  public void rawDeleteRange(BackOffer backOffer, ByteString startKey, ByteString endKey) {
```

Comment:
maybe add some notes, the caller may want to know
- [startKey, endKey] or
- [startKey, endKey) or
- (startKey, endKey] or
- (startKey, endKey)

Reply:
fixed